flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTest.java

```diff
@@ -2025,9 +2009,8 @@ private void runJobFailureWhenTaskExecutorTerminatesTest(
                                         return CompletableFuture.completedFuture(Acknowledge.get());
                                     })
                             .setHeartbeatJobManagerFunction(
-                                    heartbeatConsumerFunction.apply(
-                                            jobMasterGateway,
-                                            taskManagerUnresolvedLocation.getResourceID()))
+                                    (ignoredResourceId, ignoredAllocationSlotReport) ->
+                                            FutureUtils.completedVoidFuture())
```

Comment:
this is the default behavior of the builder; so we don't even need to set a function at all.

Reply:
Good point. I updated it 👍